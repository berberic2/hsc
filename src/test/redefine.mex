test/redefine.hsc (3,13): Warning 59: redefined tag <SEPP>
test/redefine.hsc (11,13): Warning 59: redefined tag <HUGO>
test/redefine.hsc (14,1): Bad style 67: recommended tag <HTML> missing
test/redefine.hsc (14,1): Bad style 67: recommended tag <HEAD> missing
test/redefine.hsc (14,1): Warning 15: required tag <TITLE> missing
