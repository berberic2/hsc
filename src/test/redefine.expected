sepp1: sepp1
sepp2: sepp2


hugo-open(inside)hugo-close


hugo-simple
